src/Native/configure.cmake

```diff
@@ -234,6 +234,22 @@ check_cxx_source_compiles(
 
 check_cxx_source_compiles(
     "
+    #include <netinet/tcp.h>
+    int main() { int x = TCP_ESTABLISHED; return x; }
+    "
+    HAVE_TCP_H_TCPSTATE_ENUM
+)
+
+check_cxx_source_compiles(
+    "
+    #include <netinet/tcp_fsm.h>
+    int main() { int x = TCPS_ESTABLISHED; return x; }
+    "
+    HAVE_TCP_FSM_H
+)
+
```

Comment:
It can be tested something like:

```cmake
check_symbol_exists(
    TCP_ESTABLISHED
    "netinet/tcp.h"
    HAVE_TCP_H_TCPSTATE_ENUM)

check_symbol_exists(
    HAVE_TCP_FSM_H
    "netinet/tcp_fsm.h"
    HAVE_TIOCGWINSZ)
```